docker/api/Dockerfile

```diff
@@ -17,9 +19,9 @@ COPY . .
 
 ADD https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem ./rds-combined-ca-bundle.pem
 
-RUN chown -R 1001:appgroup /app
+RUN chown -R ${UID}:appgroup /app
 
-USER 1001
+USER UID
```

Comment:
I forgot, didn't we need to revert this commit on the adapter?

Reply:
yes but i figured out what was the problem (missing a default for circle to use)